client/components/domains/register-domain-step/index.jsx

```diff
@@ -560,6 +561,21 @@ const RegisterDomainStep = React.createClass( {
 					severity = 'info';
 				}
 				break;
+			case 'tld_in_maintenance':
+				tldIndex = domain.lastIndexOf( '.' );
+				if ( tldIndex !== -1 ) {
+					message = this.translate(
+						'Domains ending with {{strong}}%(tld)s{{/strong}} are undergoing maintenance. They will be available again soon.',
```

Comment:
Maybe we can use a different copy here. It's not important the tld is in maintenance to the user. 

How about something like "Domains ending with TLD are temporarily unavailable. Please check back shortly."?

(not sure about the 2nd)